tsl/test/isolation/specs/deadlock_recompress_chunk.spec.in

```diff
@@ -95,5 +95,5 @@ step "recompress_chunks_start" {
 		      
 # Since the locking order is different, we locking before reindex we should one one lock on the index and
 # one lock on the chunk from different processes.
-permutation "recompress_insert_rows" "lock_after_decompress" "recompress_chunks_start" "query_start" "unlock_after_decompress"
```

Comment:
What does the parenthesis do here? I don't see anything in the isolation test documentation.

Reply:
https://github.com/postgres/postgres/blob/f2c52eeba919a1b191f60445001371bd7c53aaa9/src/test/isolation/README#L151

Reply:
So apparently this doesn't exist in 13.2 https://github.com/postgres/postgres/blob/REL_13_2/src/test/isolation/README or in 12.6. Is this going to be a problem?

Reply:
Yes, it was added recently in all branches (including PG11), but we ignore failures in these tests prior to the latest release in each version, so it should not be a problem. It might be prudent to not include the test for early versions though, since it is guaranteed to fail with odd errors.

Reply:
Added some checks to exclude the test for earlier versions.